eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala

```diff
@@ -96,13 +96,25 @@ object ExplorerApi {
    * Query https://blockcypher.com/ to fetch block headers.
    * See https://www.blockcypher.com/dev/bitcoin/#introduction.
    */
-  case class BlockcypherExplorer() extends Explorer {
+  case class BlockcypherExplorer(socksProxy_opt: Option[Socks5ProxyParams]) extends Explorer with SupportsTor {
+    private implicit val sb = SttpUtils.backend(socksProxy_opt)
     override val name = "blockcypher.com"
     override val baseUris = Map(
       Block.TestnetGenesisBlock.hash -> uri"https://api.blockcypher.com/v1/btc/test3",
       Block.LivenetGenesisBlock.hash -> uri"https://api.blockcypher.com/v1/btc/main"
     )
 
+    // Mimic Firefox
+    private val headers = Map(
+      "User-Agent" -> "Mozilla/5.0 (Windows NT 10.0; rv:78.0) Gecko/20100101 Firefox/78.0",
+      "Accept" -> "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8",
+      "Accept-Language" -> "en-US,en;q=0.5",
+      "Connection" -> "keep-alive",
+      "Upgrade-Insecure-Requests" -> "1",
+      "Cache-Control" -> "max-age=0"
+    )
```

Comment:
Can you move this to the `Socks5ProxyParams` object and rename it `FakeFirefoxHeaders` or something like that? It can be useful to re-use it elsewhere in the future.